src/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ set(OsConfigProjectName "osconfig")
 set(OsConfigProjectLongName "Azure OSConfig")
 set(OsConfigProjectVendor "Microsoft")
 set(OsConfigProjectEmail "osconfigsupport@microsoft.com")
+set(OsConfigProjectUrl "https://github.com/Azure/azure-osconfig/")

```

Comment:
Changes to these CMakeList.txt files in core folders risk the Ubuntu 14 build. See MariusNi/OsConfig_Ubuntu14_NrpBuild locally (please do not commit anything there)

Reply:
As discussed offline, let's do your proposal here:

if (Ubuntu14)
....
else
....

Reply:
Incorporated GCC 4.8 to disable tests/adapters/platform/moduled by default